charts/fe-nextjs/templates/proxy.yaml

```diff
@@ -0,0 +1,91 @@
+{{- if .Values.staticProxy.enabled }}
```

Comment:
I do not get the point here, why a next.js application should proxy something? how do we use it?

Reply:
See https://github.com/casavo/lisa/blob/master/helm/values/production.yaml#L33-L49